src/domain/enrolment/enrolmentForm/ValidationSchema.tsx

```diff
@@ -41,24 +41,43 @@ export default Yup.object().shape({
       const validateSumOfSizePair = (
         sizePair: number,
         schema: Yup.NumberSchema,
-        fieldLimitValidationMessage: string,
-        totalLimitValidationMessage: string
+        totalMinLimitValidationMessage: string,
+        fieldMaxLimitValidationMessage: string,
+        totalMaxLimitValidationMessage: string
       ): Yup.NumberSchema => {
-        // Validate a single field against the max size.
+        // Validate a single field against the total min and max sizes.
         // The used validation error message will be the same for both the fields.
         // This is also preventing negative param.max to occur in validation.
-        if (!sizePair || sizePair < 0 || sizePair > maxGroupSize) {
-          return schema.max(maxGroupSize, (param) => ({
-            max: param.max,
-            key: totalLimitValidationMessage,
-          }));
+        if (
+          !sizePair ||
+          sizePair < 0 ||
+          sizePair < minGroupSize ||
+          sizePair > maxGroupSize
+        ) {
+          return (
+            schema
+              // Min-limit is current a gap to minimum group size.
+              .min(
+                sizePair != null ? minGroupSize - sizePair : minGroupSize,
+                (param) => ({
```

Comment:
minor but params not used and can be deleted :)

Reply:
Removed.